packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -200,8 +200,45 @@ const validWrapperTypes = new Set([
   ClassComponent,
   HostComponent,
   ForwardRef,
+  // Normally skipped, but used when there's more than one root child.
+  HostRoot,
 ]);
 
+function getChildren(parent: Fiber) {
```

Comment:
Nit - we could explicitly flow type the return value; `Array<ReactTestInstance | string>`. 

Also I see that this was copy-pasted from the `getChildren` method below. Wish there was a docblock, it's not very clear to me why this works the way it does. But that could be for the future.

Reply:
We don't always Flow-type internal functions because those change more often. I figured that since we still have an outer annotation on the public API, we can leave this one inferred. If it returns something bad the annotation on the calling method should catch it.